src/common/event-utils/index.js

```diff
@@ -24,4 +24,35 @@ function handleUpDownArrowsKeydown(e, callback) {
     handleKeydown([38, 40], e, callback);
 }
 
-module.exports = { handleActionKeydown, handleEscapeKeydown, handleUpDownArrowsKeydown };
+const handlers = [];
```

Comment:
I think you should nest all of this code inside of a `resizeEvent` object, and export that. The `observer` name conflicts a bit with how we use it in `propertyObserver`. It's also a bit strange having these generic names in the top scope of this module which holds lots of utils.

Reply:
Should I make the changes in the other PR?

Reply:
Personally I think the event util changes should be part of this PR since the need arose from pagination, but I'm not sure if the others wanted you to keep it separate.